pipelines/broad/rna_seq/test_inputs/Plumbing/SM-HFCB1_100ng_.65X_A3.hg19.fastqs.plumbing.json

```diff
@@ -11,12 +11,12 @@
   "RNAWithUMIsPipeline.platform_unit": "barcode1",
   "RNAWithUMIsPipeline.read_group_name": "RG1",
 
-  "RNAWithUMIsPipeline.starIndex": "gs://broad-gotc-test-storage/rna_seq/hg19/STAR2.7.10a_genome_hg19_noALT_noHLA_noDecoy_v19_oh145.tar.gz",
-  "RNAWithUMIsPipeline.gtf": "gs://broad-gotc-test-storage/rna_seq/hg19/gencode.v19.genes.v7.collapsed_only.patched_contigs.gtf",
+  "RNAWithUMIsPipeline.starIndex": "gs://broad-references/hg19/v0/star/STAR2.7.10a_genome_hg19_noALT_noHLA_noDecoy_v19_oh145.tar.gz",
+  "RNAWithUMIsPipeline.gtf": "gs://broad-references/hg19/v0/annotation/gencode.v19.genes.v7.collapsed_only.patched_contigs.gtf",
   "RNAWithUMIsPipeline.ref": "gs://gcp-public-data--broad-references/hg19/v0/Homo_sapiens_assembly19.fasta",
   "RNAWithUMIsPipeline.refIndex": "gs://gcp-public-data--broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai",
   "RNAWithUMIsPipeline.refDict": "gs://gcp-public-data--broad-references/hg19/v0/Homo_sapiens_assembly19.dict",
-  "RNAWithUMIsPipeline.refFlat": "gs://broad-gotc-test-storage/rna_seq/hg19/Homo_sapiens_assembly19.refFlat.txt",
-  "RNAWithUMIsPipeline.ribosomalIntervals": "gs://broad-gotc-test-storage/rna_seq/hg19/Homo_sapiens_assembly19.rRNA.interval_list",
-  "RNAWithUMIsPipeline.exonBedFile": "gs://broad-gotc-test-storage/rna_seq/hg19/gencode.v19.hg19.insert_size_intervals_geq1000bp.bed"
+  "RNAWithUMIsPipeline.refFlat": "gs://broad-references/hg19/v0/annotation/Homo_sapiens_assembly19.refFlat.txt",
+  "RNAWithUMIsPipeline.ribosomalIntervals": "gs://broad-references/hg19/v0/annotation/Homo_sapiens_assembly19.rRNA.interval_list",
+  "RNAWithUMIsPipeline.exonBedFile": "gs://broad-references/hg19/v0/annotation/gencode.v19.hg19.insert_size_intervals_geq1000bp.bed"
```

Comment:
These look good but I should also double check that the geq1000bp.bed file is sound (as I promised @nikellepetrillo a couple weeks ago…)